jinahub/indexers/searcher/compound/FaissPostgresIndexer/README.md

```diff
@@ -96,8 +78,17 @@ All the others will fail and emit warning.
 To avoid this, you will need to split the Flow into two Flows, one for storage, and one for search, and use different polling methods.
 This is shown in the integration tests as well.
 
+### Via dump
+
+Alternatively, you can use the dump-reload method.
+
+See [docs](https://docs.jina.ai/advanced/experimental/indexers/#dump-and-rolling-update)
+
+Check [integration tests](https://github.com/jina-ai/executors/tree/main/tests/integration/psql_dump_reload) for an example on how to use it.
```

Comment:
I suggest adding a README.md at https://github.com/jina-ai/executors/tree/main/tests/integration/psql_dump_reload. So that the uses have a better idea of how this works.

Reply:
Ok but that's outside the scope of this PR. I will add that in another PR. Any feedback on this one?